js/plugins/vertexai/src/imagen.ts

```diff
@@ -53,9 +59,50 @@ export const imagen2 = modelRef({
       output: ['media'],
     },
   },
+  version: 'imagegeneration@006',
   configSchema: ImagenConfigSchema,
 });
 
+export const imagen3 = modelRef({
+  name: 'vertexai/imagen3',
+  info: {
+    label: 'Vertex AI - Imagen3',
+    versions: ['imagen-3.0-generate-001'],
+    supports: {
+      media: false,
+      multiturn: false,
+      tools: false,
+      systemRole: false,
+      output: ['media'],
+    },
+  },
+  version: 'imagen-3.0-generate-001',
+  configSchema: ImagenConfigSchema,
```

Comment:
Have you confirmed that Imagen3 has the same config schema? 1 -> 2 was pretty different.

Reply:
Once I get allowlisted, I'll take another pass through. The way I interpret the docs, everything seems to be standardized across all of the models now, but they do support different criteria on the various options (e.g. different aspect ratios, languages, etc), some params are n/a to certain models, etc.

Imagen2 also got upgraded to v006, so there will probably be some changes applicable there as well.